Bricks/template/decay.h

```diff
@@ -134,72 +136,68 @@ static_assert(is_same<const int, rmref<const int&&>>::value, "");
-static_assert(is_same<std::tuple<std::string>, rmconstref<std::tuple<const std::string>>>::value, "");
-static_assert(is_same<std::tuple<std::string>, rmconstref<std::tuple<const std::string&>>>::value, "");
-static_assert(is_same<std::tuple<std::string>, rmconstref<std::tuple<std::string&>>>::value, "");
-static_assert(is_same<std::tuple<std::string>, rmconstref<std::tuple<std::string&&>>>::value, "");
+static_assert(is_same<std::tuple<std::string>, decay<std::tuple<std::string>>>::value, "");
+static_assert(is_same<std::tuple<std::string>, decay<std::tuple<const std::string>>>::value, "");
+static_assert(is_same<std::tuple<std::string>, decay<std::tuple<const std::string&>>>::value, "");
+static_assert(is_same<std::tuple<std::string>, decay<std::tuple<std::string&>>>::value, "");
+static_assert(is_same<std::tuple<std::string>, decay<std::tuple<std::string&&>>>::value, "");
 
-static_assert(is_same<std::tuple<std::string>, rmconstref<const std::tuple<std::string>&>>::value, "");
-static_assert(is_same<std::tuple<std::string>, rmconstref<std::tuple<const std::string>>>::value, "");
-static_assert(is_same<std::tuple<std::string>, rmconstref<const std::tuple<const std::string>>>::value, "");
-static_assert(is_same<std::tuple<std::string>, rmconstref<const std::tuple<const std::string&>>>::value, "");
-static_assert(is_same<std::tuple<std::string>, rmconstref<const std::tuple<const std::string&>&>>::value, "");
-static_assert(is_same<std::tuple<std::string>, rmconstref<std::tuple<const std::string&>&&>>::value, "");
+static_assert(is_same<std::tuple<std::string>, decay<const std::tuple<std::string>&>>::value, "");
+static_assert(is_same<std::tuple<std::string>, decay<std::tuple<const std::string>>>::value, "");
+static_assert(is_same<std::tuple<std::string>, decay<const std::tuple<const std::string>>>::value, "");
+static_assert(is_same<std::tuple<std::string>, decay<const std::tuple<const std::string&>>>::value, "");
+static_assert(is_same<std::tuple<std::string>, decay<const std::tuple<const std::string&>&>>::value, "");
+static_assert(is_same<std::tuple<std::string>, decay<std::tuple<const std::string&>&&>>::value, "");
 
 // YO DAWG! I HEARD YOU LIKE TUPLES!
 static_assert(
-    is_same<std::tuple<std::tuple<std::string>>, rmconstref<std::tuple<std::tuple<const std::string>>>>::value,
-    "");
+    is_same<std::tuple<std::tuple<std::string>>, decay<std::tuple<std::tuple<const std::string>>>>::value, "");
+static_assert(is_same<std::tuple<std::tuple<std::string>>, decay<std::tuple<std::tuple<std::string&>>>>::value,
+              "");
 static_assert(
-    is_same<std::tuple<std::tuple<std::string>>, rmconstref<std::tuple<std::tuple<std::string&>>>>::value, "");
+    is_same<std::tuple<std::tuple<std::string>>, decay<std::tuple<std::tuple<const std::string&>>>>::value, "");
+static_assert(is_same<std::tuple<std::tuple<std::string>>, decay<std::tuple<std::tuple<std::string&&>>>>::value,
+              "");
+
```

Comment:
WTF?! I thought, it supposed to format strings, but not add additional vertical whitespaces.